core/components/_helpers/pagination.js

```diff
@@ -37,7 +37,13 @@ export function changePageIfAppropiate(rawNextPage, total, perPage, handlerFn) {
   const pageCount = pagesFromItems(total, perPage)
   const nextPageExists = nextPage > 0 && nextPage <= pageCount
 
-  if (nextPageExists) handlerFn(nextPage)
+  if (nextPageExists) {
+    if (handlerFn) {
+      handlerFn(nextPage)
+    } else {
+      console.error('You must define an onPageChange function.')
```

Comment:
🤔 why though?

Should we tackle this with propTypes instead?

Reply:
This is more of a helper function check. Mostly to check if we don't pass the function from the prop to the helper function.